EpiAware/src/EpiLatentModels/manipulators/ConcatLatentModels.jl

```diff
@@ -113,13 +113,13 @@ Generate latent variables by concatenating multiple latent models.
     @assert sum(dims)==n "Sum of dimensions must be equal to the dimension of the latent variables"
 
     @submodel final_latent, latent_aux = _concat_latents(
-        latent_models.models, 1, [], [], dims, latent_models.no_models)
+        latent_models.models, 1, Real[], [], dims, latent_models.no_models)
```

Comment:
Is forcing the array to have an abstract eltype 100% necessary here? Given the performance implication from [here](https://turinglang.org/docs/tutorials/docs-13-using-turing-performance-tips/#ensure-that-types-in-your-model-can-be-inferred)

Reply:
I don't know but I think this should be its own issue. This was just done to get benchmarks to run without error.

Reply:
See #345. We can use it as a test case as if it improves performance the benchmarks should flag that.